Advanced-Recruit-Training/src/BatteryStateOfChargeService.cpp

```diff
@@ -16,17 +26,20 @@ BatteryStateOfChargeService::~BatteryStateOfChargeService()
 
 double BatteryStateOfChargeService::totalAmpHoursUsed() const
 {
-    return 0.0;
+    return totalAmpHoursUsed_;
 }
 
 bool BatteryStateOfChargeService::isCharging() const
 {
-    return false;
+   if (batteryData_.current < 0)
+       return true;
+   else
+       return false;
```

Comment:
Still follow style guidelines and have braces, even if they are one-line bodies for the if/else statements.